disparate-flies/WorkforceManagement
# Comp delete

Please structure your pull requests this way:

- **Purpose of PR**: User can delete a computer as long as it is NOT associated to an employee past or present

- **Issue number this request impacts** : #16 

- **Description of what was changed**: 
1. Created the DeleteConfirm View
2. GET and POST were created for the delete method
3. added custom error message to delete if computer belongs to an employee

- **Thorough list of steps to test**
1. checkout master
2. git fetch
3. cd into my branch (CompDelete)
4. type start WorkforceManagement.sln inside your terminal, it will open up in Visual Studio
5. open the Computer Controller inside the file
6. press green play button in the navbar of VS
7. terminal will open telling you to go into browser and enter localhost:5000
8. click on the computer tab inside the navbar
9. You should see a list of all computers by their Id, Manufacturer, and Make
10. you should not be able to click on the words Edit and Details since they are not being used and have been turned to text
11. click on the Delete hyperlink
12. if the computer is associated to an employee, you will get a custom error message saying "You can not delete a computer that belongs to an employee."
13. go back to this link in your browser (http://localhost:5000/Computer), and you should be able to create a new computer
14. Once this computer is created, it will not be associated to an employee. This is the computer that should allow deletion.
15. A DeleteConfirm Popup will appear and you need to press Delete

if you don't do this you will be
**FIRED**
